frontend/projects/dipa-frontend/src/app/modules/gantt/templates-view/templates-view.component.ts

```diff
@@ -0,0 +1,251 @@
+
+  vm$: Observable<any>;
+
+  timelineData = [];
+
+  timelinesSubscription;
+  activatedRouteSubscription;
+  updateTemplateSubscription;
+
+  selectedTimelineId: number;
+  selectedProjectApproachId: number;
+  selectedOperationTypeId: number;
+  selectedOperationTypeName: string;
+  selectedProjectApproachName: string;
+
+  selectedTemplatesList = [];
+  selectedTemplatesIdList: any[];
+  standardTemplatesList = [];
+  selectedStandardTemplateIndex: number;
+  projectApproachesList: any;
+
+  standardName;
+
+  constructor(public templatesViewControlsService: TemplatesViewControlsService,
+              public ganttControlsService: GanttControlsService,
+              private timelinesService: TimelinesService,
+              private operationTypesService: OperationTypesService,
+              private projectApproachesService: ProjectApproachesService,
+              public activatedRoute: ActivatedRoute,
+              private templateService: TemplatesService) {  }
+
+  ngOnInit(): void {
+    this.selectedStandardTemplateIndex = null;
+
+    this.projectApproachesSubscription = this.projectApproachesService.getProjectApproaches()
+    .subscribe((data) => {
+      this.projectApproachesList = data;
+    });
+
+    this.activatedRouteSubscription = this.activatedRoute.params.subscribe(param => {
+      this.selectedTimelineId = param.id;
+
+      this.timelinesSubscription = this.timelinesService.getTimelines()
+        .subscribe((data) => {
+
+          this.timelineData = data;
+          this.setData();
+
+          this.selectedOperationTypeId = this.timelineData.find(item => item.id === Number(this.selectedTimelineId)).operationTypeId;
+          this.selectedProjectApproachId = this.timelineData.find(item => item.id === Number(this.selectedTimelineId)).projectApproachId;
+
+          this.operationTypesSubscription = this.operationTypesService.getOperationTypes()
+          .subscribe((resOperation) => {
+
+            this.selectedOperationTypeName = resOperation.find(item => item.id === Number(this.selectedOperationTypeId)).name;
+          });
+
+          // this.selectedProjectApproachName =  this.projectApproachesList
+          //       .find(item => item.id === Number(this.selectedProjectApproachId)).name;
+
+        });
+    });
+
+  }
+
+  ngOnDestroy(): void {
+    this.activatedRouteSubscription?.unsubscribe();
+    this.timelinesSubscription?.unsubscribe();
+    this.projectApproachesSubscription?.unsubscribe();
+    this.operationTypesSubscription?.unsubscribe();
+    this.updateTemplateSubscription?.unsubscribe();
+  }
+
+  setData(): void {
+    this.vm$ = forkJoin([
+      this.timelinesService.getTimelines(),
+      this.templateService.getTemplatesForTimeline(this.selectedTimelineId)
+    ])
+    .pipe(
+      map(([timelinesData, templatesData]) => {
+
+        this.selectedTemplatesList = [];
+
+        this.selectedTemplatesList.push(templatesData.find(t => t.name === 'aktuell'));
```

Comment:
auch lieber über Konstante in der Komponente: private CURRENT_TEMPLATE = 'aktuell'; oder so